ext/pcntl/config.m4

```diff
@@ -43,6 +44,12 @@ if test "$PHP_PCNTL" != "no"; then
     ]),,,
     [#include <sys/wait.h>])
 
+  AC_CHECK_FUNCS([syscall],,,
+    [#include <unistd.h>])
```

Comment:
As a more permissive check, we could use AC_CHECK_DECLS instead of AC_CHECK_FUNCS.

Or indeed we can add it into the common AC_CHECK_FUNCS, `unistd.h` is already included in `configure.ac`. There's an additional Linux guard check and at least glibc, musl and uclibc declare it as a function from `unistd.h`, so shuld be safe for most users.

Reply:
I went with the latter change (common AC_CHECK_FUNCS).